### ISSUE TITLE
Remove EOVS versions

### ISSUE THREAD

**Issue by jdwnetapp** [NetAppDocs/ontap-tools-vmware-vsphere#44]
### Page URL

https://docs.netapp.com/us-en/ontap-tools-vmware-vsphere/deploy/concept_installation_workflow_for_existing_users_of_ontap_tools.html

### Page title

Deployment workflow for existing users of ONTAP tools

### Summary

This page still mentions SRA 2.0, 3.0, 4.0.  IMO, we need to remove all instructions for very old, EOVS versions from this page.

### Public issues must not contain sensitive information

- [X] This issue contains no sensitive information.

**Comment by netapp-jani** [comment]
@jdwnetapp 
Thank you for contacting us with your feedback. After your inputs are researched internally, we will determine how best to address it and update the content at the earliest.

Regards,
Jani

**Comment by netapp-jani** [comment]
@jdwnetapp 
Thank you for your valuable input.
We discussed this internally and decided to keep the older versions as there still could be a small set of customers who are using the older versions. However, we have changed the instruction on page [Deployment workflow for existing users of ONTAP tools](https://docs.netapp.com/us-en/ontap-tools-vmware-vsphere/deploy/concept_installation_workflow_for_existing_users_of_ontap_tools.html) and have instructed the user to contact support for upgrades.
Hope this was useful.
